sdk/core/azure-core-amqp/vendor/azure-uamqp-c/src/amqpvalue.c

```diff
@@ -1072,17 +1072,18 @@ AMQP_VALUE amqpvalue_create_string(const char* value)
         size_t length = strlen(value);
         size_t malloc_size = length + 1;
 
-        // If the result of malloc_size is zero it means it had a type overflow (size_t is an unsigned
-        // type). It is very unlikely but could happen.
+        // If the result of malloc_size is zero it means it had a type overflow (size_t is an unsigned type).
+        // It is very unlikely but could happen.
         if (malloc_size == 0)
         {
-          LogError("Invalid string size exceeded max allocation");
-          result = NULL;
+            LogError("Invalid string size exceeded max allocation");
+            result = NULL;
         }
         else
         {
-          result = REFCOUNT_TYPE_CREATE(AMQP_VALUE_DATA);
+            result = REFCOUNT_TYPE_CREATE(AMQP_VALUE_DATA);
         }
+
```

Comment:
If we're not moving clang-format, can we disable it for this directory?
For example, these changes that I am highlighting in my comment, they are not part of the commit that you are porting. This PR has more changes than the uamqp commit.
Unless these indentation changes are already present in the uamqp repo, and you are also porting them along with the fix, then I think it is good - so in the future, when we are porting more stuff, the diffs are smaller, and also it is easier to compare two using the file diff tool. If this is the case, then I would welcome even more of these. Maybe in a separate PR, so it is easier to follow (but don't bother removing them from this PR).
But if they are not present in the parent repo, I suggest to undo them from this PR.

Reply:
The changes to amqpvalue.c are there because the original port of the fixes from uAMQP-c was sloppy, this resolves inconsistencies between the two repos.

As a part of porting the current fix, I ran beyond compare on the two repos looking for all differences and noticed these inconsistencies with the parent repo.

Reply:
Oh, and we don't run clang-format on this directory.  

We only look at .cpp and .hpp files when we run clang-format.